Dockerfile

```diff
@@ -33,14 +29,16 @@ WORKDIR /app
 COPY Gemfile Gemfile.lock ./
 RUN gem install bundler && bundle install --jobs 20 --retry 5
 
+# Do the same for node packages, allowing them to be cached
 RUN npm update -g
+COPY package.json package-lock.json ./
+RUN npm install
 
 # Copy the main application.
 COPY . ./
 
-RUN yarn install
-RUN npm rebuild node-sass
```

Comment:
So we do not need rebuild were neither, right?

Reply:
yeah the issue was that locally built node-sass was overwriting the one built in the docker container. adding node_modules to .dockerignore is meant to fix this.